frontend/express/public/javascripts/countly/countly.helpers.js

```diff
@@ -1647,31 +1647,83 @@
     /** Adds column selector to data table
      * @param {object} dtable - data table jquery object
      * @param {object} config - configuration for disabling columns
-     * @param {object} config.disabled - object for disabled column numbers. Optional. If nothing set, you can disable all columns. Example: {"1":true,"2":true}
-     * @param {object} config.hidden - object for default hidden columns. Example: {"1":true,"2":true} If user has changed it, it gets overwritten by data stored in local storage.
+     * @param {object} config.disabled - object for disabled column names. Optional. If nothing set, you can disable all columns. Example: {"name1":true,"name2":true}
+     * @param {object} config.visible -Default visible columns. If none set - first N are taken to not exceed max column count. If user has changed it, it gets overwritten by data stored in local storage.
      * @param {number} config.maxCol  - max column count. If not set - max == all columns.
      * @param {string} tableName - table name. Used to create name for storage. need to be unique for every table.
      * 
      *  Example:
-     *   CountlyHelpers.addColumnSelector(dtable,{"disabled:"{"1":true,"2":true},"hidden":{},maxCol:6},"myTableName");
+     *   CountlyHelpers.addColumnSelector(dtable,{"disabled:"{"name1":true,"name2":true},maxCol:6},"myTableName");
      *  Safe way would be adding in "fnInitComplete" function:
      *
      *  "fnInitComplete": function(oSettings, json) {
      *      $.fn.dataTable.defaults.fnInitComplete(oSettings, json);
-     *      CountlyHelpers.addColumnSelector(this, {"disabled":{"0":true,"8":true}, "maxCol":5 }, "viewsTable");
+     *      CountlyHelpers.addColumnSelector(this, {"disabled":{"name1":true,"name2":true}, "maxCol":5 }, "viewsTable");
      * },
      */
     CountlyHelpers.addColumnSelector = function(dtable, config, tableName) {
         config = config || {};
-        var settings = store.get(tableName + "HiddenDataTableColumns") || {};
-        if (Object.keys(settings).length === 0 && config && config.disabled) { // we don't have stored value
-            settings = config.hidden;
-            store.set(tableName + "HiddenDataTableColumns", settings);
+        config.disabled = config.disabled || {};
+        config.visible;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=5166101)